quartz/components/scripts/popover.inline.ts

```diff
@@ -48,15 +48,16 @@ async function mouseEnterHandler(
 
   if (!response) return
   const contentType = response.headers.get("Content-Type")
```

Comment:
we should split by `;` here instead

```suggestion
  const [contentType] = response.headers.get("Content-Type").split(";")
```